src/dfcx_scrapi/builders/response_messages.py

```diff
@@ -0,0 +1,250 @@
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import re
+import logging
+from typing import List, Dict, Union, Any
+
+from google.cloud.dialogflowcx_v3beta1.types import ResponseMessage
+from google.protobuf import struct_pb2
+from dfcx_scrapi.builders.builders_common import BuildersCommon
+
+# logging config
+logging.basicConfig(
+    level=logging.INFO,
+    format="%(asctime)s %(levelname)-8s %(message)s",
+    datefmt="%Y-%m-%d %H:%M:%S",
+)
+
+
+class ResponseMessageBuilder(BuildersCommon):
+    """Base Class for CX ResponseMessage builder."""
+
+    _proto_type = ResponseMessage
+    _proto_type_str = "ResponseMessage"
+
+
+    def __str__(self) -> str:
+        """String representation of the proto_obj."""
+        self._check_proto_obj_attr_exist()
+
+        if self.proto_obj.text:
+            resp_type = "text"
+            resp_msg = self.proto_obj.text.text
+        elif self.proto_obj.payload:
+            resp_type = "payload"
+            proto_struct = self.proto_obj.payload
+            resp_msg = "\n\t".join([
+                f"{k}: {v}" for k, v in proto_struct.items()
+            ])
+        elif self.proto_obj.conversation_success:
+            resp_type = "conversation_success"
+            proto_struct = self.proto_obj.conversation_success.metadata
+            resp_msg = "\n\t".join([
+                f"{k}: {v}" for k, v in proto_struct.items()
+            ])
+        elif self.proto_obj.output_audio_text:
+            if self.proto_obj.output_audio_text.text:
+                resp_type = "output_audio_text - text"
+                resp_msg = self.proto_obj.output_audio_text.text
+            elif self.proto_obj.output_audio_text.ssml:
+                resp_type = "output_audio_text - ssml"
+                resp_msg = self.proto_obj.output_audio_text.ssml
+        elif self.proto_obj.live_agent_handoff:
+            resp_type = "live_agent_handoff"
+            proto_struct = self.proto_obj.live_agent_handoff.metadata
+            resp_msg = "\n\t".join([
+                f"{k}: {v}" for k, v in proto_struct.items()
+            ])
+        elif self.proto_obj.play_audio:
+            resp_type = "play_audio"
+            resp_msg = self.proto_obj.play_audio.audio_uri
+        elif self.proto_obj.telephony_transfer_call:
+            resp_type = "telephony_transfer_call"
+            resp_msg = self.proto_obj.telephony_transfer_call.phone_number
+
+        return (
+            f"Response Type: {resp_type}\nMessage:\n\t{resp_msg}"
+        )
+
+
+    def show_response_message(self):
+        """Show the proto_obj information."""
+        self._check_proto_obj_attr_exist()
+        print(self)
+
+
+    def create_new_proto_obj(
```

Comment:
@MRyderOC I'd like to see each of the sub-sections for `text`, `payload`, etc. broken out into their own creation methods. This will make for easier debugging down the road.

I'm not going to let it block the release, but I wanted to make a note of it so you know to fix later on.